std/complex.d

```diff
@@ -352,25 +340,12 @@ if (isFloatingPoint!T)
     ref Complex opOpAssign(string op, C)(const C z)
         if (op == "/" && is(C R == Complex!R))
     {
-        import std.math : fabs;
-        if (fabs(z.re) < fabs(z.im))
-        {
-            immutable ratio = z.re/z.im;
-            immutable denom = z.re*ratio + z.im;
-
-            immutable temp = (re*ratio + im)/denom;
-            im = (im*ratio - re)/denom;
-            re = temp;
-        }
-        else
-        {
-            immutable ratio = z.im/z.re;
-            immutable denom = z.re + z.im*ratio;
+        immutable y = re * re + im * im;
+        immutable p = z.re * re + z.im * im;
+        immutable q = z.im * re - z.re * im;
```

Comment:
IMHO this should be:

```
immutable y = z.re * z.re + z.im * z.im;
immutable p = z.re * re + z.im * im;
immutable q = z.re * im - z.im * re;
```

Reply:
Yes, I tested the function was working locally, but didn't transpose it correctly into opOpAssign.